src/sample_submission_data.R

```diff
@@ -0,0 +1,9 @@
+## Script that generates sample submission data to pass validation checks
+library("arrow") # for parquet writing
+source(here::here('src/sim_model_output.R'))
+# Function found in /src to generate data frame
+df_sim <- sim_model_output()
+
+# Note: make sure to point to the correct path for your data, defaults to folder 'model-output'
+arrow::write_parquet(df_sim, here::here("auxiliary-data/example-files/2024-10-02-umass-validtestsubmission.parquet"))
+hubValidations::validate_model_data(here::here(), "2024-10-02-umass-validtestsubmission.parquet")
```

Comment:
If I understand this correctly, whoever uses this script should manually change `auxiliary-data/example-files/` to `model-outputs/umass-validtestsubmission/`?

Reply:
If the user wants to validate the data locally via hubValidations I believe that is correct. 
When submitting to the hub, the submission file also needs to to be in their `model-output` team folder ([Hubverse model-output](https://hubverse.io/en/latest/user-guide/model-output.html)).
As this script generates a sample submission file for reference, it seems reasonable to keep it out of such a place and in auxiliary data.

Reply:
That's what I was assuming. It was the impetus for https://github.com/hubverse-org/hubValidations/issues/130.